### ISSUE TITLE
Apps crash on Debian 11 (Chrome OS, WSL2?) due to old Wayland: wl_proxy_marshal_flags

### ISSUE THREAD

**Issue by cbm755** [flathub/com.riverbankcomputing.PyQt.BaseApp#247]
I'm getting crashes when I tried my app on Chrome OS, where it used to work https://gitlab.com/plom/plom/-/issues/3131  
As soon as I close any windows, dialog, popup, etc, I crashes with messages about `wl_proxy_marshal_flags` and "Maybe the compositor crashed".

I think I have tracked this down to  https://bugreports.qt.io/browse/QTBUG-114635

Basically, the host OS (Debian 11 I think) ships an older version of Wayland (1.18) and we need 1.19.  Is this fixable in `com.riverbankcomputing.PyQt.BaseApp`?   For example, by building Wayland as that Qt bug discusses?

I have tested locally that `pip install PyQt6==6.5.0 PyQt6-Qt6==6.5.0` works ok Chrome OS.  

I also reported this to Riverbank Computing: https://www.riverbankcomputing.com/pipermail/pyqt/2023-November/045587.html

**Comment by JakobDev** [comment]
No, this is not fixable. Wayland is part of your System. We can't ship a other Version with the Flatpak. This BaseApp also in contrast to the PyPI Package only PyQt and not Qt itself. It uses the Qt Version from the [KDE Runtime](https://invent.kde.org/packaging/flatpak-kde-runtime). You should report this Bug there, as it will affect all Qt Apps. Maybe they can find a Way to patch this or disable Wayland on older Systems, but you should not get too much hope. As I workaround I recommend to use `--socket=x11` instead of `--socket=wayland` and `--socket=fallback-x11` to force using the XServer.
